backend/user_management/main.py

```
@@ -54,9 +116,16 @@
     return user
 
 
+<<<<<<< HEAD
```

Ruff (0.8.2)

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected an indented block after function definition

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

Trailing comma missing

(COM812)